lib/params.js

```diff
@@ -27,3 +27,28 @@ module.exports.file = (req, res, next, param) => {
         next();
     });
 };
+
+/**
+ * Validate the "type" parameter
+ *
+ * @param {Object} req HTTP request object
+ * @param {Object} res HTTP response object
+ * @param {function} next Next function in the route
+ * @param {String} param Parameter from the URL
+ */
+
+module.exports.type = (req, res, next, param) => {
+    schemas.type.validate(param, (error, value) => {
```

Comment:
Any reason you are using the callback form of validate or just preference?

Reply:
To keep with the style of the others.